libs/tenant_size_model/src/lib.rs

```diff
@@ -0,0 +1,349 @@
+use std::borrow::Cow;
+use std::collections::HashMap;
+
+/// Pricing model or history size builder.
```

Comment:
https://github.com/neondatabase/neon/pull/2714/commits/f082c1090ee892666fa36813b6e7b3bae6217c34

Reply:
Still remains the `pricing` in libs/tenant_size_model/README.md header... I am inclined to keep it, because it should be revealed somewhere that this is related to pricing attempts.